pages/api/me/password.js

```diff
@@ -34,17 +29,15 @@ import RecordNotFound from '@klaudsol/commons/errors/RecordNotFound';
 export default withSession(handleRequests({ put }));
 
 async function put(req, res) { 
- try{
-
-    await assert({
-     loggedIn: true,
-    }, req);
-
-    const { session_token } = req.session;
```

Comment:
1.) Why did we remove the try-catch block?
2.) Why did we remove the assertion? I'm guessing the `loggedIn` assertion doesn't work anymore for JWT. In that case, `assert` needs to be refactored to check cases when a bearer token is present.

Reply:
Better yet,

Kindly use assertUserCan at the beginning of all API endpoints.

https://github.com/klaudsol/klaudsol-commons/blob/main/src/lib/Permissions.js#L78